Verify fails for v0.2.7

I tried installing Chainsaw with `verify: true` but it failed. Here is a link to the [pipeline](https://github.com/cloudnative-pg/charts/actions/runs/10164925829/job/28111519575?pr=341), if that's useful.

GitHub Action configuration:

```
jobs:
  test-cluster-standalone:
    runs-on: ubuntu-22.04
    steps:
      - name: Install Cosign
        uses: sigstore/cosign-installer@v3.1.1

      - name: Install Chainsaw
        uses: kyverno/action-install-chainsaw@v0.2.7
        with:
          verify: true
```

Error Details:

```
INFO: Custom version 'v0.2.7' requested
INFO: Downloading Chainsaw version 'v0.2.7'...
      https://github.com/kyverno/chainsaw/releases/download/v0.2.7/chainsaw_linux_amd64.tar.gz
Error: getting Rekor public keys: updating local metadata and targets: error updating to TUF remote mirror: invalid key
remote status:{
	"mirror": "https://tuf-repo-cdn.sigstore.dev/",
	"metadata": {
		"root.json": {
			"version": 9,
			"len": 6766,
			"expiration": "12 Sep 24 06:53 UTC",
			"error": ""
		},
		"snapshot.json": {
			"version": 150,
			"len": 2304,
			"expiration": "13 Aug 24 16:07 UTC",
			"error": ""
		},
		"targets.json": {
			"version": 9,
			"len": 5478,
			"expiration": "12 Sep 24 06:13 UTC",
			"error": ""
		},
		"timestamp.json": {
			"version": 206,
			"len": 721,
			"expiration": "02 Aug 24 16:05 UTC",
			"error": ""
		}
	}
}
main.go:74: error during command execution: getting Rekor public keys: updating local metadata and targets: error updating to TUF remote mirror: invalid key
remote status:{
	"mirror": "https://tuf-repo-cdn.sigstore.dev/",
	"metadata": {
		"root.json": {
			"version": 9,
			"len": 6766,
			"expiration": "12 Sep 24 06:53 UTC",
			"error": ""
		},
		"snapshot.json": {
			"version": 150,
			"len": 2304,
			"expiration": "13 Aug 24 16:07 UTC",
			"error": ""
		},
		"targets.json": {
			"version": 9,
			"len": 5478,
			"expiration": "12 Sep 24 06:13 UTC",
			"error": ""
		},
		"timestamp.json": {
			"version": 206,
			"len": 721,
			"expiration": "02 Aug 24 16:05 UTC",
			"error": ""
		}
	}
}
Error: Process completed with exit code 1.
```

--- a/README.md
+++ b/README.md
@@ -64,7 +64,7 @@ jobs:
     name: Install Chainsaw
     steps:
       - name: Install Cosign
-        uses: sigstore/cosign-installer@v3.1.1
+        uses: sigstore/cosign-installer@v3.6.0
       - name: Install Chainsaw
         uses: kyverno/action-install-chainsaw@v0.1.0
         with:

